app/mixins/check-authenticated-route-mixin.js

```diff
@@ -0,0 +1,8 @@
+import Mixin from '@ember/object/mixin';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+export default Mixin.create(AuthenticatedRouteMixin, {
+  authenticationRoute: 'login',
```

Comment:
What is this for?

Reply:
If we are going to have different login routes / templates for ```admin``` and ```user``` for example, than we  can add authenticationRoute property to specify exact route where user must be redirected